src/test/java/net/deckserver/ChatParserTest.java

```diff
@@ -46,7 +46,10 @@ public void testParseCard() throws Exception {
         test = "A link to [Theo Bell] and [Theo Bell (G2)] and [Theo Bell (ADV)] and [Theo Bell (G6)] should all work";
         modified = ChatParser.parseText(test);
 
-        //2023/05/13; this test does not work:
```

Comment:
As far as I can tell this test does work, hopefully removing this comment is helpful, happy to put it back if it's not.